lib/stellar_spectrum/services/send_payment.rb

```diff
@@ -12,9 +12,10 @@ def self.call(
       seeds:,
       redis_url:,
       horizon_url:,
+      logger:,
       force_transaction_source: nil,
       force_sequence_number: nil,
-      force_lock: false
+      force_lock: false,
```

Comment:
If this isn't valid (since it's not a hash, it's named args in Ruby), you'll need to make it:

```suggestion
      force_lock: false
```

Reply:
Yep, Travis complained about it. Thanks!